packages/primitives/src/Uint.ts

```
@@ -0,0 +1,305 @@
+
+/**
+ * Parses ether units to wei (U256)
+ * @param value - The ether value as a string.
+ */
+export const parseEtherToU256 = (value: string): Effect.Effect<U256, Error> =>
+  Effect.gen(function*(_) {
+    const wei = parseEther(value);
+    // Manual validation
+    const maxValue = (1n << 256n) - 1n;
+    if (wei > maxValue || wei < 0n) {
+      return yield* _(Effect.fail(new Error(`Value out of range for U256`)));
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)